components/modules/MentorCard/variants/Desktop.tsx

```diff
@@ -0,0 +1,211 @@
+import {} from "twin.macro";
```

Comment:
Looks like you're not importing anything from `twin.macro` here.

Reply:
Fixed